### PR TITLE
Better Windows Installation Via Scoop

### PR DESCRIPTION
**Pull request by dwu359**
Better installation of dlp-cli via scoop. Avoids needing WSL or git fetching the manifest files locally or editing path env vars.

Steps:
1. scoop bucket add scoop-dlp-cli https://github.com/DSGT-DLP/scoop-dlp-cli.git
2. scoop install scoop-dlp-cli/dlp-cli

### CODE REVIEW

**Comment by dwu359** [comment]
Updates are also automatically handled by goreleaser, works in basically the same way as brew taps.

**Review by farisdurrani — DISMISSED** [review]

**Comment by karkir0003** [comment]
whats scoop @dwu359

**Comment by karkir0003** [comment]
make sure to updatw readme

File: .goreleaser.yaml [hunk 1]
```diff
@@ -55,21 +55,14 @@ brews:
       zsh_completion.install "completions/dlp-cli.zsh" => "_dlp-cli"
       fish_completion.install "completions/dlp-cli.fish"
 
-winget:
+scoops:
```
Comment (karkir0003) [inline .goreleaser.yaml:58]: this isnt a typo? @dwu359

**Review by karkir0003 — COMMENTED** [review]
please address minor comment @dwu359

File: .goreleaser.yaml (hunk 1 above)
Comment (dwu359) [reply .goreleaser.yaml:58]: No, scoops fits our use case much better than winget rn, so I'm probably just going to delete anything winget related in this pr or in the future

**Comment by dwu359** [comment]
https://github.com/DSGT-DLP/dlp-cli/assets/54150946/02f9afd8-3991-446b-818d-dd92f9de2cea

video

**Review by farisdurrani — APPROVED** [review]

**Pull request merged**